### Caroline Scalf - Playgrounds Per Neighborhood
Playgrounds are an important aspect of healthy development for children. To determine the best neighborhood in this regard, I looked at the amount of parks per neighborhood.

In [16]:
import pandas as pd
playground = pd.read_csv('playgrounds.csv')[["name","neighborhood"]]
playground.head(20)

name              neighborhood
0             Able Long Playground                 Beechview
1         Albert Graham Playground          Crawford-Roberts
2                Alpine Playground         Central Northside
3                 Alton Playground                 Beechview
4                 Ammon Playground         Bedford Dwellings
5             Arlington Playground         South Side Slopes
6             Armstrong Playground          South Side Flats
7               Arsenal Playground       Lower Lawrenceville
8         August WIlson Playground          Crawford-Roberts
9            Banksville Playground                Banksville
10               Baxter Playground            Homewood North
11              Bon Air Playground                   Bon Air
12      Boundary Street Playground           Central Oakland
13     Brighton Heights Playground          Brighton Heights
14            Brookline Playground                 Brookline
15           Bud Hammer Playground                 Hazelwood
16              Burgwin Playground                 Hazelwood
17  Camp David Lawrence Playground       Squirrel Hill South
18             Catalono Playground             Spring Garden
19             Chadwick Playground  Lincoln-Lemington-Belmar

In [27]:
len(playground)

125

This gives me the total amount of playgrounds in Pittsburgh. Therefore, on my final graph, I'll have 125 different parks to display.

In [28]:
playground["neighborhood"].unique()

array(['Beechview', 'Crawford-Roberts', 'Central Northside',
       'Bedford Dwellings', 'South Side Slopes', 'South Side Flats',
       'Lower Lawrenceville', 'Banksville', 'Homewood North', 'Bon Air',
       'Central Oakland', 'Brighton Heights', 'Brookline', 'Hazelwood',
       'Squirrel Hill South', 'Spring Garden', 'Lincoln-Lemington-Belmar',
       'Windgap', 'Troy Hill', 'Westwood', 'Perry South', 'Homewood West',
       'Allegheny Center', 'Strip District', 'Stanton Heights',
       'Crafton Heights', 'West Oakland', 'East Carnegie',
       'Duquesne Heights', 'East Liberty', 'Esplen', 'Fairywood',
       'Highland Park', 'Upper Lawrenceville', 'Fineview',
       'Regent Square', 'Garfield', 'Greenfield', 'South Oakland',
       'Allentown', 'Hays', 'Elliott', 'Morningside', 'Terrace Village',
       'Larimer', 'Carrick', 'Central Lawrenceville', 'Lincoln Place',
       'Manchester', 'Marshall-Shadeland', 'Sheraden', 'Beltzhoover',
       'Shadyside', 'Mount Washington', 'Oakwo

This is the amount of unique neighborhoods featured in this data set.

In [26]:
import pandas as pd
playground = pd.read_csv('playgrounds.csv')
new_playground = pd.DataFrame

SyntaxError: invalid syntax (<ipython-input-26-dce2fb8fe2dd>, line 3)

In [24]:
import pandas as pd
import numpy as np
import geopandas
playground = pd.read_csv('playgrounds.csv')
playground.sort_values("name")["neighborhood"].plot.bar

<bound method PlotAccessor.bar of <pandas.plotting._core.PlotAccessor object at 0x7f9ff87dbeb0>>